client/src/extension.ts

```diff
@@ -1,20 +1,23 @@
-import * as path from 'path';
-import { workspace, ExtensionContext, commands, languages, Disposable } from 'vscode';
+import { join } from 'path';
+import { workspace, ExtensionContext, languages, Disposable } from 'vscode';
 
 import {
     LanguageClient,
     LanguageClientOptions,
     ServerOptions,
-    TransportKind,
+    TransportKind, ClientCapabilities
 } from 'vscode-languageclient';
 import OMTLinkProvider from './omtLinkProvider';
+import { ENETRESET } from 'constants';
 
 let client: LanguageClient;
 
 export function activate(context: ExtensionContext) {
+    const test: ClientCapabilities = {};
```

Comment:
This isn't used anywhere I think, also its name is `test`. Can it go?

Reply:
done in the upcoming tslint commit